src/components/modal/Modal.scss

```diff
@@ -144,8 +144,23 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: fixed;
+            top: 0;
+            left: 0;
+            z-index: 2;
```

Comment:
Is this `z-index` necessary? If we use `absolute` positioning, can this be removed?

Reply:
Managed to use `position: absolute` remove the z-index after adjusting some of the other styles. 

Basically, the `.modal-content` effectively has a `margin-top: 64px` and `margin-bottom: 60px` which correspond to the height of the `.modal-header-container` and the `.modal-actions` footer respectively.